lib/libyear_bundler/models/ruby.rb

```diff
@@ -108,6 +108,11 @@ def newest_version_sequence_index
 
       def release_date(version)
         v = all_stable_versions.detect { |ver| ver['version'] == version }
+
+        if v.nil?
```

Comment:
I have a fix for what might be causing this to be `nil`. I'll push it up. We can leave this here just in case.